src/sqlfluff/dialects/dialect_duckdb.py

```diff
@@ -449,6 +503,16 @@ class SelectStatementSegment(ansi.SelectStatementSegment):
     match_grammar = ansi.SelectStatementSegment.match_grammar.copy(
         insert=[Ref("QualifyClauseSegment", optional=True)],
         before=Ref("OrderByClauseSegment", optional=True),
+        replace_terminators=True,
+        terminators=[
+            Ref("SetOperatorSegment"),
+            Ref("WithNoSchemaBindingClauseSegment"),
+            Ref("WithDataClauseSegment"),
+            Sequence("ON", "CONFLICT"),
+            Ref.keyword("RETURNING"),
```

Comment:
```suggestion
            "RETURNING"
```
You can specify keywords directly as strings if no options are needed

Reply:
Ok, good to know, thanks.  I'll update it.

FYI, I copied that part from: https://github.com/sqlfluff/sqlfluff/blob/004ed009e08e84405488afe7367053f3fb4388b3/src/sqlfluff/dialects/dialect_postgres.py#L1785